### PR TITLE
Es-829648: Convert_Word_to_Image

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Subpage added for convert Word to Image.

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Blazor.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    //Open an existing Word document.
+    using (WordDocument document = new WordDocument(sourceStreamPath, FormatType.Docx))
+    {
+        //Instantiation of DocIORenderer for Word to Image conversion
+        using (DocIORenderer render = new DocIORenderer())
+        {
+            Stream imageStream = document.RenderAsImages(0, ExportImageFormat.Jpeg);
+            //Reset the stream position.
+            imageStream.Position = 0;
+            return (MemoryStream)imageStream;
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Blazor.md:275]: MemoryStream imageStream = (MemoryStream)document.RenderAsImages(0, ExportImageFormat.Jpeg);
//Reset the stream position.
imageStream.Position = 0;
//Download PDF file in the browser.
await JS.SaveAs("WordToImage.Jpeg", imageStream.ToArray());

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Mac.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Step 1: Create a new .NET Core console application project.
+
+![Create .NET Core console application in Visual Studio](Mac-images/CreateProject.png)
+
+Step 2: Select the project version.
+
+![Select Project version](Mac-images/selectprojectverion.png)
+
+Step 3: Install the [Syncfusion.DocIORenderer.Net.Core ](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core/) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).Install the Nuget Package on Mac please refer this [Link](https://learn.microsoft.com/en-us/visualstudio/mac/nuget-walkthrough?view=vsmac-2022)
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Mac.md:23]: Add proper space after full stop and before stating new sentance.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Window-forms.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+{% highlight c# tabtitle="C#" %}
+
+//Load the existing Word document 
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    //Convert the first page of the Word document into an image.
+    Image image = document.RenderAsImages(0, ImageType.Bitmap);
+    //Save the image as jpeg.
+    image.Save("WordToImage.Jpeg");
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Window-forms.md:94]: image.Save("WordToImage.Jpeg", ImageFormat.Jpeg);

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-WPF.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    //Convert the first page of the Word document into an image.
+    System.Drawing.Image image = document.RenderAsImages(0, ImageType.Bitmap);
+    //Save the image as jpeg.
+    image.Save("WordToImage.Jpeg");
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-WPF.md:78]: image.Save("WordToImage.Jpeg", ImageFormat.Jpeg);

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-ASP-NET-MVC.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+documentation: UG
+---
+
+# Convert Word document to Image in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in ASP.NET MVC**.
+
+## Steps to convert Word document to Image in C#:
+
+Step 1: Create a new ASP.NET Web Application  project.
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-ASP-NET-MVC.md:15]: updtaed

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-ASP-NET.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in ASP.NET Web Forms**.
+
+## Steps to convert Word document to Image in C#:
+
+Step 1: Create a new ASP.NET Web application project.
+
+![Create ASP.NET Web application in Visual Studio](ASP-NET_images/CreateProjectforConversion.png)
+
+Step 2: Select the Empty project
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-ASP-NET.md:19]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-WPF.md (hunk 4 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-WPF.md:78]: update

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Mac.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Mac.md:23]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Blazor.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Blazor.md:275]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-MAUI.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+  <td>
+    <b>Summary</b>
+  </td>
+  </tr>
+  <tr>
+  <td>
+    {{'[.NET MAUI Project](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Convert-Word-to-image/.NET-MAUI/Convert-Word-Document-to-Image)'| markdownify }}
+  </td>
+  <td>
+    {{'[SaveService.cs](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Convert-Word-to-image/.NET-MAUI/Convert-Word-Document-to-Image/SaveServices)'| markdownify }}
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-MAUI.md:129]: UPDATED

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**